### ISSUE TITLE
Display counters in text output about count of found dependencies for the check type

### ISSUE THREAD

**Issue by andrew-demb** [shipmonk-rnd/composer-dependency-analyser#166]
For now, output looks like this (from the readme)

```
Found shadow dependencies!
(those are used, but not listed as dependency in composer.json)

  • nette/utils
    e.g. Nette\Utils\Strings in app/Controller/ProductController.php:24 (+ 6 more)

Found unused dependencies!
(those are listed in composer.json, but no usage was found in scanned paths)

  • nette/utils

(scanned 13970 files in 2.297 s)
```

I suggest adding a counter to the end of the report about the check type (shadow deps, unused deps, etc) - this will be helpful in case of long list of issued deps

Example:
```
Found shadow dependencies!
(those are used, but not listed as dependency in composer.json)

  • nette/utils
    e.g. Nette\Utils\Strings in app/Controller/ProductController.php:24 (+ 6 more)

Count: 1

Found unused dependencies!
(those are listed in composer.json, but no usage was found in scanned paths)

  • nette/utils

Count: 1

(scanned 13970 files in 2.297 s)
```

**Comment by JanTvrdik** [comment]
I like the idea, but I think it would be better just say change

```
Found shadow dependencies!
```

to 

```
Found 3 shadow dependencies!
```

**Comment by andrew-demb** [comment]
I'm OK with both ways.

The proposal about writing to the end of the output was about minifying scrolling for users.

**Comment by janedbal** [comment]
Available since [1.7.0](https://github.com/shipmonk-rnd/composer-dependency-analyser/releases/tag/1.7.0)
